django_auth_ldap/backend.py

```diff
@@ -177,7 +177,10 @@ def has_module_perms(self, user, app_label):
         return False
 
     def get_all_permissions(self, user, obj=None):
-        return self.get_group_permissions(user, obj)
+        # Black have a problem when uses "|"
```

Comment:
Can you explain the problem? With the current version of black, the following code looks fine.